bridges/otelslog/handler.go

```diff
@@ -124,10 +129,46 @@ func NewHandler(options ...Option) *Handler {
 
 // Handle handles the passed record.
 func (h *Handler) Handle(ctx context.Context, record slog.Record) error {
-	// TODO: implement.
+	h.logger.Emit(ctx, h.convertRecord(record))
 	return nil
 }
 
+func (h *Handler) convertRecord(r slog.Record) log.Record {
+	var record log.Record
+	record.SetTimestamp(r.Time)
```

Comment:
From https://opentelemetry.io/docs/specs/otel/logs/data-model/#field-observedtimestamp

> This field SHOULD be set once the event is observed by OpenTelemetry.

Could we set the `ObservedTimestamp` to `time.Now`?

Reply:
Yeah, it's not clear to me what constitutes "OpenTelemetry". It's it the bridge, bridge API, or SDK?

I'm happy to add it here if there's no objections.

Reply:
This is a good question.

> For events that originate in OpenTelemetry (e.g. using OpenTelemetry Logging SDK) this timestamp is typically set at the generation time and is equal to Timestamp. For events originating externally and collected by OpenTelemetry (e.g. using Collector) this is the time when OpenTelemetry’s code observed the event measured by the clock of the OpenTelemetry code.

I feel like OpenTelemetry here refers to the API and SDK (whatever is in the https://github.com/open-telemetry/opentelemetry-go)

https://opentelemetry.io/docs/specs/otel/logs/#direct-to-collector

This PR looks like a part of the `logging library` to me.

---

I think we can keep the current implementation.